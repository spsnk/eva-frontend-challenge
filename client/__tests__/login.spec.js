import React from "react"
import {create} from "react-test-renderer"
import LoginForm from "../src/components/login"

describe("LoginForm", () => {
  test("Renders correctly", () => {
    const component = create(<LoginForm />).toJSON();
    expect(component).toMatchSnapshot();
  })
})