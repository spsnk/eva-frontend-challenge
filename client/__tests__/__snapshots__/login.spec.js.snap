// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginForm Renders correctly 1`] = `
<div
  className="container"
  style={
    Object {
      "position": "relative",
    }
  }
>
  <form
    className="mb-5 mx-auto d-flex flex-column"
    onSubmit={[Function]}
    style={
      Object {
        "maxWidth": "600px",
      }
    }
  >
    <small
      className="form-text"
    >
      Enter your login Information:
    </small>
    <div
      className="form-group"
    >
      <div
        className="row"
      >
        <label
          className="form-label col-form-label col-sm-2"
          htmlFor="username"
        >
          E-mail:
        </label>
        <div
          className="col"
        >
          <input
            className="form-control"
            id="username"
            name="username"
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            placeholder="somebody@somewhere.com"
            required={true}
            type="email"
            value=""
          />
        </div>
      </div>
    </div>
    <div
      className="form-group"
    >
      <div
        className="row"
      >
        <label
          className="form-label col-form-label col-sm-2"
          htmlFor="password"
        >
          Password:
        </label>
        <div
          className="col"
        >
          <input
            className="form-control"
            id="password"
            name="password"
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            placeholder="Password"
            required={true}
            type="password"
            value=""
          />
        </div>
      </div>
    </div>
    <button
      className="ml-auto btn btn-light"
      disabled={false}
      type="submit"
    >
      Login
    </button>
  </form>
</div>
`;
